packages/components/src/local/organisms/channel-messages-list/index.tsx

```diff
@@ -7,16 +7,15 @@ import styles from './styles.module.scss'
 
 /* Import Components */
 import ChannelMessage from '../../molecules/channel-message'
-import ChannelRFIMessage from '../../molecules/channel-rfi-message'
 import ForcesInChannel from '../../molecules/forces-in-channel'
 import { Box } from '@material-ui/core'
 // import collateMessages from './helpers/collate-messages'
 import { INFO_MESSAGE_CLIPPED } from '@serge/config'
-import { MessageChannel, MessageCustom, ForceRole, MessageDetailsFrom } from '@serge/custom-types'
+import { MessageChannel, MessageCustom } from '@serge/custom-types'
 import { formatTurn } from '@serge/helpers'
 
 /* Render component */
-export const ChannelMessagesList: React.FC<PropTypes> = ({ messages, icons, colors, onMarkAllAsRead, onRead, onUnread, onChange, isRFIManager, isUmpire, turnPresentation }: PropTypes) => {
+export const ChannelMessagesList: React.FC<PropTypes> = ({ messages, icons, colors, onMarkAllAsRead, onRead, onUnread, isUmpire, turnPresentation }: PropTypes) => {
```

Comment:
@lilitkarapetyan - the changes in the above line seem significant - the props are quite different.

Can you explain why they are different please?

Reply:
https://github.com/serge-web/serge/pull/1164/files#diff-b80b0eacec650b3d35312300f89caa98001505c1653717fb4465c928f15075d6L36
If you check this part, you will see that code was removed, so as we don't using ChannelRFIMessage component more, we had a unused vars `onChange`, `isRFIManager` which ones removed from props